client/my-sites/sidebar/use-site-menu-items.js

```diff
@@ -106,6 +110,13 @@ const useSiteMenuItems = () => {
 	if ( shouldShowGlobalSidebar ) {
 		return globalSidebarMenu();
 	}
+	if ( shouldShowGlobalSiteViewSidebar ) {
+		return globalSiteViewSidebarMenu( {
+			siteDomain,
+			shouldShowAddOns: shouldShowAddOnsInFallbackMenu,
+			showSiteMonitoring: isAtomic,
+		} );
+	}
```

Comment:
This needs gating behind classic view:

Something like:

```
   const adminInterface = useSelector( ( state ) =>
		getSiteOption( state, selectedSiteId, 'wpcom_admin_interface' )
	);
	if ( shouldShowGlobalSiteViewSidebar  && adminInterface === 'wp-admin' ) {
}
```

(Or push it up into useGlobalSidebar - but only for the site view variable as we require classic view for that)

Reply:
https://github.com/Automattic/wp-calypso/pull/87477/commits/7bfdd0f7da435a931fba7aee17054f4eebea25a9